common/models/application.js

```diff
@@ -71,13 +72,11 @@ function generateKey(hmacKey, algorithm, encoding) {
 
 module.exports = function(Application) {
   // Workaround for https://github.com/strongloop/loopback/issues/292
-  Application.definition.rawProperties.created.default =
```

Comment:
This change looks suspicious to me, could you please explain which `eslint` rule is violated by this line?